code/game/objects/items/weapons/handcuffs.dm

```diff
@@ -158,3 +158,93 @@ var/last_chew = 0
 
 /obj/item/weapon/handcuffs/cyborg
 	dispenser = 1
+
+
+/obj/item/weapon/handcuffs/ziptie
+	name = "zipties"
+	desc = "Sturdy, and reliable plastic zipties for binding the wrists."
+	gender = PLURAL
+	icon = 'icons/obj/items.dmi'
+	breakouttime = 600 //One minute, because they're not made of steel, therefore easier to slip out of.
+	icon_state = "ziptie"
+	cuff_sound = 'sound/weapons/cablecuff.ogg'
+	var/singular_name = "ziptie"
+	var/amount = 6
+	var/max_amount = 6
+
+/obj/item/weapon/handcuffs/ziptie/New(var/loc, var/amount=null)
+	..()
+	if (amount)
+		src.amount=amount
+	return
+
+/obj/item/weapon/handcuffs/ziptie/Del()
+	if (src && usr && usr.machine==src)
+		usr << browse(null, "window=stack")
+	..()
+
+/obj/item/weapon/handcuffs/ziptie/examine()
+	set src in view(1)
+	..()
+	usr << "There are [src.amount] [src.singular_name]\s in the stack."
+	return
+
+/obj/item/weapon/handcuffs/ziptie/proc/use(var/amount)
+	src.amount-=amount
+	if (src.amount<=0)
+		src = null //dont kill proc after del()
+	return
+
+/obj/item/weapon/handcuffs/ziptie/proc/add_to_stacks(mob/usr as mob)
+	var/obj/item/weapon/handcuffs/ziptie/oldsrc = src
+	src = null
+	for (var/obj/item/weapon/handcuffs/ziptie/N in usr.loc)
+		if (N==oldsrc)
+			continue
+		if (!istype(N, oldsrc.type))
+			continue
+		if (N.amount>=N.max_amount)
+			continue
+		oldsrc.attackby(N, usr)
+		usr << "You add new [N.singular_name] to the stack. It now contains [N.amount] [N.singular_name]\s."
+		if(!oldsrc)
+			break
+
+/obj/item/weapon/handcuffs/ziptie/attack_hand(mob/user as mob)
+	if (user.get_inactive_hand() == src)
+		var/obj/item/weapon/handcuffs/ziptie/F = new src.type( user, 1)
+		F.copy_evidences(src)
+		user.put_in_hands(F)
+		src.add_fingerprint(user)
+		F.add_fingerprint(user)
+		use(1)
+		if (src && usr.machine==src)
+			spawn(0) src.interact(usr)
+	else
+		..()
+	return
+
+/obj/item/weapon/handcuffs/ziptie/attackby(obj/item/W as obj, mob/user as mob)
+	..()
+	if (istype(W, src.type))
+		var/obj/item/weapon/handcuffs/ziptie/S = W
+		if (S.amount >= max_amount)
+			return 1
+		var/to_transfer as num
+		if (user.get_inactive_hand()==src)
+			to_transfer = 1
+		else
+			to_transfer = min(src.amount, S.max_amount-S.amount)
+		S.amount+=to_transfer
+		if (S && usr.machine==S)

```

Comment:
From here, remove the following:
```
		if (S && usr.machine==S)
			spawn(0) S.interact(usr)
		src.use(to_transfer)
		if (src && usr.machine==src)
			spawn(0) src.interact(usr)
```

This is a relic of someone being a copy-paste warrior with metal sheets.